locales/en/messages.po

```diff
@@ -3145,6 +3153,10 @@ msgstr "<0>Yes, you’ll still owe rent after sending the letter because our sta
 msgid "pay rent, rent, can't pay rent, june rent, june 1"
 msgstr "pay rent, rent, can't pay rent, june rent, june 1"
 
+#: frontend/lib/laletterbuilder/letter-builder/choose-letter.tsx:18
+msgid "{0}"
+msgstr "{0}"
+
```

Comment:
Yeah I think this is not an actual translation—you may need to rework how you are internationalizing the letter choices